iphone/Classes/TiAppiOSProxy.m

```diff
@@ -451,25 +451,23 @@ - (void)registerUserNotificationSettings:(id)args
         break;
       }
       }
-      // Handle additional iOS 12+ enums
-      if ([TiUtils isIOSVersionOrGreater:@"12.0"]) {
-        switch ([TiUtils intValue:thisTypeRequested]) {
-        case UNAuthorizationOptionCriticalAlert: // USER_NOTIFICATION_TYPE_CRITICAL_ALERT
-        {
-          types |= UNAuthorizationOptionCriticalAlert;
-          break;
-        }
-        case UNAuthorizationOptionProvisional: // USER_NOTIFICATION_TYPE_PROVISIONAL
-        {
-          types |= UNAuthorizationOptionProvisional;
-          break;
-        }
-        case UNAuthorizationOptionProvidesAppNotificationSettings: // USER_NOTIFICATION_TYPE_PROVIDES_APP_NOTIFICATION_SETTINGS
-        {
-          types |= UNAuthorizationOptionProvidesAppNotificationSettings;
-          break;
-        }
-        }
+
+      switch ([TiUtils intValue:thisTypeRequested]) {
```

Comment:
merge into the switch statement above - no point having two switch statements for the same value now

Reply:
Done. Thanks!